keylayout/ft5x06_ts.kl

```diff
@@ -25,12 +25,7 @@
 # OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN
 # IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# Hardware keys
```

Comment:
Why dropping hardware keys?

Reply:
Not dropping ! remapped n removed the comments
Check that commit

Reply:
You removed POWER, VOLUME_UP/DOWN

Reply:
Done.

Reply:
Please squash/amend your commits in a proper way.

Reply:
okay